### DISCUSSION TITLE
Customize Node internal IP

### DISCUSSION THREAD

**Discussion by BoDING1007** [rancher-sandbox/rancher-desktop#5624] [Q&A]
Looks like in Rancher Desktop 1.9.1(Windows), the node internal IP is fixed now and do not change after restart. This is good improvement already, my question is that Can I customize it? In my case, I would like to fix the ip in 172.30.2.2.
![image](https://github.com/rancher-sandbox/rancher-desktop/assets/24220535/28ce20aa-13f4-4315-991f-f19792f656e3)

**Comment by jandubois** [comment]
Which platform is this, Windows or macOS/Linux?

**Reply by BoDING1007** [reply]
Windows
